com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/DoFCycleOfConfusion.compute

```diff
@@ -0,0 +1,56 @@
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+#include "Packages/com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl"
```

Comment:
it is circle of confusion, not cycle of confusion :), please rename the file

Reply:
Ah, yes, of course you are right! It will be renamed in the next PR, because I did not see the comment before the merge :)

Reply:
I have rename it, no worry